### DISCUSSION TITLE
Git hub copilot not working in Pycharm

### DISCUSSION THREAD

**Discussion by Shaik23540007** [community/community#140088] [GitHub Education]
### Hi everyone,

i am not able to use git hub copilot in pycharm , it is showing like **please try login github and try again**

![Screenshot 2024-09-29 115000](https://github.com/user-attachments/assets/a7589528-01ea-4f41-be5f-bd801a81238a)

**Comment by iPriyanshu19** [comment]
Logout first then login again and check.
Give permission to use copilot in PyCharm from GitHub.

**Reply by Shaik23540007** [reply]
yeah ,i tried bro its not working.

**Comment by davevad93** [comment]
Hi @Shaik23540007 ,

Did you get `Copilot` with the `Student Developer Pack`? If so, what's the status of your application?

If the label is green and states "`Congrats! Your request for @Shaik23540007  was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @Shaik23540007 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "All offers" section and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

**Reply by Shaik23540007** [reply]
Still it is on Green

**Reply by davevad93** [reply]
@Shaik23540007 that means your benefits aren't available yet that's why you can't use `Copilot`. You have to wait for the status to change to `Congrats! Your request for @Shaik23540007 was approved and your benefits are now available `, then you'll be able to use `Copilot` and `Student Developer Pack` benefits.

**Reply by Shaik23540007** [reply]
Okay, I'll check after it’s approved. I haven't logged in yet, even though it's showing that message.
